sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/TestHelper.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.appconfiguration;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.provider.Arguments;
+
+import static com.azure.core.test.TestBase.getHttpClients;
+
+class TestHelper {
+    static final String DISPLAY_NAME_WITH_ARGUMENTS = "{displayName} with [{arguments}]";
+    private static final String AZURE_TEST_SERVICE_VERSIONS = "AZURE_APPCONFIG_SERVICE_VERSIONS";
+    private static final String serviceVersionFromEnv = Configuration.getGlobalConfiguration().get(AZURE_TEST_SERVICE_VERSIONS);
+    private static final String AZURE_TEST_SERVICE_VERSIONS_VALUE_ALL = "ALL";
```

Comment:
This can be declared once in `TestBase.java` in core-test module as ALL can be common for all libraries.